docs/docs/guides/database-gui.md

```diff
@@ -4,8 +4,17 @@ A short guide on connecting [pgAdmin](https://www.pgadmin.org/) to Immich.
 
 :::note
 
-- In order to connect to the database the immich_postgres container **must be running**.
-- The passwords and usernames used below match the ones specified in the example `.env` file. If changed, please use actual values instead.
+In order to connect to the database the immich_postgres container **must be running**.
+
+The passwords and usernames used below match the ones specified in the example `.env` file. If changed, please use actual values instead.
+
+**Optional:** To connect to the database **outside** of your Docker's network:
+
+- Expose port 5432 in your `docker-compose.yml` file.
+- Edit the PostgreSQL [`pg_hba.conf`](https://www.postgresql.org/docs/current/auth-pg-hba-conf.html) file.
```

Comment:
I don't think you need to do this

Reply:
Even the database is in the container?

Reply:
Are you using the default compose setup and config? I've never needed to do this.

Reply:
> Are you using the default compose setup and config? I've never needed to do this.

Yes, everything is on default except the addition of exposing ports in the docker file

 ```
 redis:
    container_name: immich_redis
    image: redis:6.2-alpine@sha256:c5a607fb6e1bb15d32bbcf14db22787d19e428d59e31a5da67511b49bb0f1ccc
    restart: always

  database:
    container_name: immich_postgres
    image: tensorchord/pgvecto-rs:pg14-v0.1.11@sha256:0335a1a22f8c5dd1b697f14f079934f5152eaaa216c09b61e293be285491f8ee
    env_file:
      - .env
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
    volumes:
      - pgdata:/var/lib/postgresql/data
    ports:
      - 5432:5432
     
I also disabled my firewall, my computers all
192.168.X.X/24 by default. (server is 192.168.1.230 my computer is 192.168.1.153)
Until I edited the pg_hba.conf file it simply prevented me from connecting.